packages/cubejs-api-gateway/openspec.yml

```diff
@@ -173,7 +173,9 @@ components:
         members:
           type: array
           items:
-            type: "string"
+            oneOf:
+              - type: string
+              - $ref: "#/components/schemas/V1CubeMetaFolder"
```

Comment:
We need to regenerate the client, because otherwise, defining nested folders will break SQL API, MDX & DAX API, while it's not possible to deserialize such a meta response.

Thanks